### ISSUE TITLE
Notifications View - I wish the sort would indicate which sort is being used

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode-pull-request-github#6337]
Testing #6329

![Image](https://github.com/user-attachments/assets/4868f1ff-9f70-4062-b19a-726b4f442cac)

If I choose one of these, I expect there to be some indication as to which one is currently the primary sorting behavior.

**Comment by lszomoru** [comment]
Agreed, unfortunately the concept of a "checked" menu item is not currently supported through our extension API.

**Comment by gjsjohnmurray** [comment]
Perhaps this can be the catalyst for action on PR #214102 by @eamodio 🙏

**Comment by alexr00** [comment]
Nice, thanks for connecting the dots @gjsjohnmurray!
